src/lib.rs

```diff
@@ -112,7 +112,10 @@ impl CompilationHandle for CompilationHandleImpl {
 }
 
 /// If this file is not found, please refer to https://enter-tainer.github.io/typst-preview/dev.html to build the frontend.
+#[cfg(feature = "embed-html")]
 const HTML: &str = include_str!("../addons/vscode/out/frontend/index.html");
+#[cfg(not(feature = "embed-html"))]
+const HTML: &str = "HTML not embedded.";
```

Comment:
This is not a valid HTML string.

Reply:
I think that doesn't really matter, as long as it can display something strange in browser